modules/core/src/main/scala/org/tessellation/domain/snapshot/SnapshotStorage.scala

```diff
@@ -0,0 +1,19 @@
+package org.tessellation.domain.snapshot
+
+import org.tessellation.schema.snapshot.Snapshot
+import org.tessellation.schema.transaction.Transaction
+import org.tessellation.schema.{Block, SnapshotOrdinal}
+import org.tessellation.security.hash.Hash
+import org.tessellation.security.signature.Signed
+
+trait SnapshotStorage[F[_], T <: Transaction, B <: Block[T], S <: Snapshot[T, B]] {
```

Comment:
do you need to specify `T` and `B`? looks unused 🤔

Reply:
I can use wildcard `S <: Snapshot[_, _]`

Reply:
Given that we are not fetching blocks or transactions (unwrapped from within the snapshot) from this storage, the wildcards should work.

Reply:
then probably we need it parametrized @kpudlik